### DISCUSSION TITLE
New releases

### DISCUSSION THREAD

**Discussion by muraty** [celery/py-amqp#418] [General]
Hello.

What is the reason behind to not release new changes? The latest version is 5.1.1 and released more than 1 year ago. After that, there are important commits such as https://github.com/celery/py-amqp/commit/a92dd037712b5b7b1622f4f9d83157d095c90910.

Thanks,
Murat.

**Comment by Nusnus** [comment]
@auvipy do you want me to handle it or do you wish to do it yourself?

I agree it makes sense to release a new version, especially if there are important fixes waiting.

**Reply by auvipy** [reply]
I want to handle this release. hopefully this week
